consultation_analyser/consultations/jinja2/consultation-uploaded.html

```diff
@@ -0,0 +1,20 @@
+{%- from 'govuk_frontend_jinja/components/panel/macro.html' import govukPanel -%}
+{% extends "base.html" %}
+
+{% set page_title = "Consultation uploaded" %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    {{ govukPanel({
+      "titleText": "Consultation uploaded",
+      }) }}
+
+    <h2 class="govuk-heading-m">What happens next</h2>
+
+    <p class="govuk-body">The team will be notified that your consultation is ready for processing</p>
```

Comment:
Do you want to change this line to say something like "tell the i.AI team that you have uploaded your consultation"?

Given that email notification is de-prioritised as of sprint planning today.